libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/service/EvcsService.java

```diff
@@ -109,6 +104,13 @@ public void storePendingVc(VerifiableCredential credential) throws EvcsServiceEx
                                 credential.getVcString(), PENDING_RETURN, null, null)));
     }
 
+    @Tracing
```

Comment:
I wonder if this tracing is at the right level - having tracing on the EVCS client methods is quite useful, but we probably don't need it on all the private methods in this class...

Reply:
done